pkgs/development/python-modules/homf/default.nix

```diff
@@ -39,6 +44,9 @@ buildPythonPackage rec {
   # nativeCheckInputs = [ pytestCheckHook ];
   # pytestFlagsArray = [ "-m 'not network'" ];
 
+  # (Ab)using `callPackage` as a fix-point operator, so tests can use the `homf` drv
+  passthru.tests = callPackage ./tests.nix { };
```

Comment:
Can you explain what these "tests" do?

Reply:
That's of no use since a FOD won't be re-run if the version of homf changes. Please remove.